author/tests.py

```diff
@@ -1,3 +1,31 @@
 from django.test import TestCase
+from .models import Author
 
-# Create your tests here.
+class AuthorModelTest(TestCase):
+    @classmethod
+    def setUpTestData(cls):
+        Author.objects.create(username="Test_author",bio_detail="This is sample bio")
+        
+    def get_author(self):
+        return Author.objects.get(id=1)
+    
+    def test_username_label(self):
+        author = self.get_author()
+        field_label = author._meta.get_field('username').verbose_name
+        self.assertEqual(field_label , 'username')
+        
+    def test_username_max_length(self):
+        author = self.get_author()
+        max_length = author._meta.get_field('username').max_length
+        self.assertEqual(max_length,50)
+        
+    def test_bio_detail_label(self):
+        author = self.get_author()
+        field_label = author._meta.get_field('bio_detail').verbose_name
+        self.assertEqual(field_label,'bio detail')
+        
+    def test_object_name(self):
+        author = self.get_author()
+        expected_name = author.username
+        self.assertEqual(expected_name, str(author))
```

Comment:
Tell me what are you testing here.

Reply:
These are the testcases for author model according to assessment doc. you have provided.

In first test and third test, I've tested if created author object has correct field label.
Second test is to check whether max_length of username field is correct.
Fourth test is to check that  __str__() returns the expected value.

Reply:
Only fourth one makes sense. You should write the test cases to check if all the author details are correct? If blog has the correct author or not. Logged out users can't edit the author, etc like this.

You should not write the test cases to check if column exist or not, they are useless.